seedcase_sprout/core/sprout_checks/check_package_properties.py

```diff
@@ -21,10 +20,10 @@ def check_package_properties(properties: dict, check_required=True) -> dict:
             fields. Defaults to True.
 
     Returns:
-        `properties`, if all checks passed.
+        `properties` if all checks pass.
 
     Raises:
-        FailedCheckError: If at least one check failed.
+        ExceptionGroup: A group of `CheckError`s, one for each check that failed.
```

Comment:
```suggestion
        ExceptionGroup: A group of `CheckError`s, one error per failed check.
```
Same for other docstrings with this error :)